lib/handle-create-event.js

```diff
@@ -11,9 +11,12 @@ const createIssue = require('./create-issue')
 
 const template = readFileSync(pathResolve(__dirname, '..', 'instructions.md'), 'utf8')
 
-function handleCreateEvent (context) {
+async function handleCreateEvent (context) {
   const {ref, ref_type: refType, repository} = context.payload
+  const config = await context.config('first-timers.yml', {labels: ['first-timers-only', 'hacktoberfest'], repository: {name: repository.name}})
```

Comment:
it should be just `repository: repository.name`. The `repository` configuration is a string, not an object with a name property: https://github.com/hoodiehq/first-timers-bot/pull/138/files#diff-04c6e90faac2675aa89e2176d2eec7d8R58